### ISSUE TITLE
Add user to list of teams

### ISSUE THREAD

**Issue by TotallyInformation** [obumnwabude/github-invite#5]
Hi, is it possible to add the invited user to a list of team names?

**Comment by obumnwabude** [comment]
Please I don't understand. How exactly do you mean? Please explain your question more

**Comment by TotallyInformation** [comment]
There is a GitHub API that lets you assign users to a team in an organisation. In the GitHub admin ui, when inviting a user to an organisation, you can also select what teams you want to add them to.

So really, this package needs to be able to do something similar.

**Comment by obumnwabude** [comment]
> There is a GitHub API that lets you assign users to a team in an organisation. In the GitHub admin ui, when inviting a user to an organisation, you can also select what teams you want to add them to.
> 
> So really, this package needs to be able to do something similar.

Yes you are right. That's a feature that would be nice for this package to have.

Out of the 4 different versions of this github-invite, the npm-package and frontend-only versions are accessed directly by the admin and take the admin's personal access token. I think it's okay to incorporate what you are requesting in these versions. That the admin selects the team in which the user they invite will join at invitation point, just as you are demanding. I will work towards this. 

As for the Node Monolithic and Firebase versions, that just provide an interface for the invitee to be invited; I don't think it's fine to add the option for the invitee to select their team at point of themselves getting invited. Do you think it is okay to ask them to choose their teams at the point the invite themselves? Let me hear your opinion please

**Comment by TotallyInformation** [comment]
Hi, thanks for the response.

We certainly wouldn't be letting invitees chose their own teams I'm afraid, that is set up front by the organisation/team administrator. We have a variety of teams who work on a variety of projects, some more sensitive than others.

**Comment by obumnwabude** [comment]
Okay, Great!

I will update the admin facing npm-package and frontend-only versions to include teams. 
Thanks for raising this issue.

Is there any other feature you'll want to see?

**Comment by TotallyInformation** [comment]
Great, thanks! Nothing else I can spot at the moment thanks.
